srcpkgs/python3-multidict/template

```diff
@@ -14,15 +13,12 @@ license="Apache-2.0"
 #changelog="https://raw.githubusercontent.com/aio-libs/multidict/master/docs/changes.rst"
```

Comment:
Any reason it's still commented out?

Reply:
Yes: I overlooked it, but it also happened to be the wrong URL. Fixed.